ui/shared/Input.qml

```diff
@@ -189,18 +189,19 @@ Item {
     }
 
     TextEdit {
-        visible: !!validationError
         id: validationErrorText
-        text: validationError
         anchors.top: inputField.bottom
         anchors.topMargin: validationErrorTopMargin
+        anchors.left: inputField.left
         anchors.right: inputField.right
+        horizontalAlignment: TextEdit.AlignHCenter
+        visible: !!validationError
```

Comment:
Are you 100% sure the error text should be centred? This component is used in quite a lot of other components.

Reply:
no! it was like this in SeedPhraseTextArea.qml line 87. I took it from there.